tracked-progresses/README.md

```diff
@@ -6,7 +6,7 @@
   "type": "Type of the document, i.e task or user. (String)",
   "userId": "The id of the user, will be applicable only for user. (String)",
   "taskId": "The id of the task, will be applicable only for task. (String)",
-  "currentlyTracked": "A boolean indicating whether the item is currently being tracked or not. (Boolean)",
+  "marked": "A boolean indicating whether the item is currently being tracked or not. (Boolean)",
```

Comment:
wasnt is monitored? marked gives a different sense

Reply:
Ok

Reply:
+1